src/resolvers/loginUserResolver.ts

```diff
@@ -1,134 +1,119 @@
-import { AuthenticationError } from "apollo-server";
-import { LoggedUserModel } from "../models/AuthUser";
-import { RoleModel } from "../models/roleModel";
-import { PermissionModel } from "../models/permissionModel";
+import { AuthenticationError } from 'apollo-server';
+import { LoggedUserModel } from '../models/AuthUser';
+import BcryptUtil from '../utils/bcrypt';
+import { generateToken } from '../utils/generateToken';
+import { validateUserLogged } from '../validations/createUser.validation';
+import { RoleModel } from '../models/roleModel';
 
 export const loggedUserResolvers: any = {
   Query: {
     async user_Logged(_: any, args: any, ctx: any) {
       const id = args.ID;
-      const upvalue = await LoggedUserModel.findById(id).populate('role');
+      const upvalue = await LoggedUserModel.findById(id);
       return upvalue;
     },
     async getUsers_Logged(_: any, args: any, ctx: any, amount: any) {
-      const users = await LoggedUserModel.find().sort({ createdAt: -1 }).limit(amount).populate({
-        path: 'role',
-        populate: {
-          path: 'permissions',
-          model: PermissionModel, 
-        },
-      });
-      return users;
+      // if (!ctx.currentUser) {
+      //   throw new AuthenticationError("You must be logged in");
+      // }
```

Comment:
Good work @buka1calvin, please remove commented-out code, and check throughout that you do not have any recurrence of commented code

Reply:
ok sir let ,me work on it